app/templates/events/view/tickets/index.hbs

```diff
@@ -0,0 +1,107 @@
+<div class="ui three statistics">
+  <div class="statistic"><div class="value">4343</div><div class="label">{{t 'Orders'}}</div></div>
+  <div class="statistic"><div class="value">0.00</div><div class="label">{{t 'Sales(US $)'}}</div></div>
+  <div class="statistic"><div class="value">1</div><div class="label">{{t 'Tickets sold'}}</div></div>
+</div>
+<br>
+<div class="row">
+  <h4 class="ui header">Order Summary&nbsp;<span>By order status</span></h4>
+  <table class="ui celled table">
+    <thead>
+      <tr>
+        <th>{{t 'Order Status'}}</th>
+        <th class="right aligned">{{t 'Total Tickets'}}</th>
+        <th class="right aligned">{{t 'Total Orders'}}</th>
+        <th class="right aligned">{{t 'Total Sales'}} (US$)</th>
+        <th>{{t 'View Reports'}}</th>
+      </tr>
+      <tr>
+        <td class="ui grey inverted segment">
+          <span>Cancelled</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui yellow inverted segment">
+          <span>Pending</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui red inverted segment">
+          <span>Expired</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui blue inverted segment">
+          <span>Placed</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui green inverted segment"><span>Completed</span></td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+    </thead>
+  </table>
+</div>
+<br>
+<div class="row">
+  <table class="ui celled table">
+    <thead>
+      <th scope="col">{{t 'Ticket Type'}}</th>
+      <th class="right aligned" scope="col">{{t 'Ticket Capacity Set'}}</th>
+      <th class="ui green inverted segment right aligned" scope="col" colspan="2">{{t 'Completed orders'}}</th>
+      <th class="ui blue inverted segment right aligned" scope="col" colspan="2">{{t 'Placed Orders'}}</th>
+      <th class="ui orange inverted segment right aligned" scope="col" colspan="2">{{t 'Pending Orders'}}</th>
+      <tr>
```

Comment:
This should be a header. Not a row.

The header alignment and positioning is confusion. And we can differentiate b/w the second level header and the text.
<img width="1203" alt="screen shot 2017-05-31 at 11 18 25 am" src="https://cloud.githubusercontent.com/assets/2404372/26617535/3cb8a0c0-45f3-11e7-84b3-be6645d866ca.png">

We need something like this
<img width="1325" alt="screen shot 2017-05-31 at 11 20 43 am" src="https://cloud.githubusercontent.com/assets/2404372/26618016/f06f1b06-45f5-11e7-9cc9-93e09d41ed8a.png">